### PR TITLE
Interview PR

### PR DESCRIPTION
**Pull request by victorFSL**
Can you take a look at this PR? I need your help 😭 my tests keep failing.

Let's get on a call so I can explain further.

### CODE REVIEW

File: cypress/interview/interview.spec.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    it("then the top menu renders", () => {
+      let topMenuHeaders = {
+        "Plans & Menu": "plans-and-menu",
+        "How it Works": "https://about.freshly.com/why-freshly",
+        "Gifts": "gifts"
+      }
+
+      for ( const [text, link] of Object.entries( topMenuHeaders ) ) {
+        cy.get('.new-nav-menu').should("contain", text)
```
Comment (jkaloya) [inline cypress/interview/interview.spec.js:16]: Testing data attributes or text is preferred to using class names or Ids. We should refrain from using those. *Documentation Link*

File: cypress/interview/interview.spec.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      let header = "Chef cooked healthy meals delivered to you"
+
+      cy.get(".home-form").within(() => {
+        cy.get("h1.home-page-section-title")
+      })
+      cy.get("form#email-form-top").within(() => {
+        cy.get('#email-top').type("test-interview@test.com")
+        cy.get("#zip-code-top").type("10005")
+      })
+      cy.get('.primary-blue-button').click()
```
Comment (jkaloya) [inline cypress/interview/interview.spec.js:31]: Looks like there multiple '.primary-blue-button' on the page. We need to specify it's the 'Get Started'

File: cypress/integration/interview/interview.spec.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      let topMenuHeaders = {
+        "Plans & Menu": "plans-and-menu",
+        "How it Works": "https://about.freshly.com/why-freshly",
+        "Gifts": "gifts"
+      }
+
+      for ( const [text, link] of Object.entries( topMenuHeaders ) ) {
+        cy.get('.new-nav-menu').should("contain", text)
+      }
+      cy.get('.new-nav-menu').should("contain", "Get $40")
```
Comment (adamesj) [inline cypress/integration/interview/interview.spec.js:18]: I think you had the right idea but might have overlooked using the function testLink below. If I may suggest, on line 13 add "Get $40": "#" then replace lines 16 and 18 with `testLink('.new-nav-menu', text, link);`

File: cypress/integration/interview/interview.spec.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    })
+
+    it("then the get started section renders correctly", () => {
+      let header = "Chef cooked healthy meals delivered to you"
```
Comment (adamesj) [inline cypress/integration/interview/interview.spec.js:22]: Double-check and make sure you don't have any spelling errors.

File: cypress/integration/interview/interview.spec.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+      cy.get(".home-form").within(() => {
+        cy.get("h1.home-page-section-title")
+      })
+      cy.get("form#email-form-top").within(() => {
+        cy.get('#email-top').type("test-interview@test.com")
+        cy.get("#zip-code-top").type("10005")
+      })
+      cy.get('.primary-blue-button').click()
```
Comment (adamesj) [inline cypress/integration/interview/interview.spec.js:31]: We can move this to line 30 to ensure we're selecting the right button.

File: cypress/integration/interview/interview.spec.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    })
+  })
+})
+
+const testLink = (selector, linkText, link) => {
```
Comment (adamesj) [inline cypress/integration/interview/interview.spec.js:36]: We can move to support folder as a helper function if we feel this will be re-used elsewhere.

**Pull request closed without merging**